### PR TITLE
activate support for gestural triple accidentals

### PR DESCRIPTION
**Pull request by rettinghaus**

### CODE REVIEW

**Comment by lpugin** [comment]
Could you rebase instead of merging? Thanks

**Comment by rettinghaus** [comment]
Ok, thought it was merged already.

**Comment by rettinghaus** [comment]
@craigsapp would you please have a look and approve this PR?

File: src/iohumdrum.cpp [hunk 1]
```diff
@@ -18937,8 +18937,8 @@ void HumdrumInput::convertNote(Note *note, hum::HTp token, int staffadj, int sta
         if (showInAccidGes) {
             if (loaccid.empty()) {
                 switch (accidCount) {
-                    // case +3: note->SetAccidGes(ACCIDENTAL_GESTURAL_ts); break;
-                    // case -3: note->SetAccidGes(ACCIDENTAL_GESTURAL_tf); break;
+                    case +3: note->SetAccidGes(ACCIDENTAL_GESTURAL_ts); break;
+                    case -3: note->SetAccidGes(ACCIDENTAL_GESTURAL_tf); break;
```
Comment (lpugin) [inline src/iohumdrum.cpp:18941]: This fails. It should be `accid` and not `note`

Comment (rettinghaus) [reply src/iohumdrum.cpp]: Obviously, yes. I leave it to Craig.

**Pull request merged**